meta.go

```diff
@@ -1,5 +1,3 @@
 package auth0
 
-// Version of this library. This value is generated automatically during the
-// release process; DO NOT EDIT.
-var Version = "latest"
+var Version = "1.3.1"
```

Comment:
Planning to switch to ship for changelog generation so maintaining the version in repo is required